.github/workflows/move-work-in-progress-prs.yml

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Move PR To Work In Progress
+on:
+  issue_comment:
+    types: [created]
+    branch:
+      - main
+
+jobs:
+  move-pr-to-wip:
+    if: contains(github.event.comment.body, '/pr mark wip') && github.event.issue.pull_request

```

Comment:
As we discussed on Discord, I'd like to make the commands a bit shorter, so people can remember them more easily: `/pr wip` and `/pr review`. (Similar to slash azp run.)